src/configurationProvider.ts

```diff
@@ -737,6 +737,11 @@ async function updateDebugSettings(event?: vscode.ConfigurationChangeEvent) {
             const exceptionFilters = {
                 skipClasses: await substituteFilterVariables(debugSettingsRoot.settings.exceptionBreakpoint.skipClasses),
             };
+
+            let asyncJDWP: string = debugSettingsRoot.settings.jdwp.async;
+            if (asyncJDWP === "auto" && vscode?.env?.appName === "Visual Studio Code - Insiders") {
```

Comment:
`vscode.env?.appName` I think the `?` after `vscode` is unnecessary?

Reply:
nice.